test/test.cc

```diff
@@ -2549,7 +2550,7 @@ TEST_F(ServerTest, StaticFileRanges) {
           .find(
               "multipart/byteranges; boundary=--cpp-httplib-multipart-data-") ==
       0);
-  EXPECT_EQ("265", res->get_header_value("Content-Length"));
+  EXPECT_EQ("266", res->get_header_value("Content-Length"));
```

Comment:
Why did you change the value to '266'?

Reply:
the request is cli_.Get("/dir/test.abcde", {{make_range_header({{1, 2}, {4, -1}})}});
this range {4, -1}

before:
ret range is "bytes 4-/5" , it was wrong because the range-end is empty(in method make_content_range_header_field).


after:
ret range is "bytes4-4/5",fix the range-end.(the purpose of the PR ) 

so.add 1 byte.

full body is:

`
----cpp-httplib-multipart-data-REkZlTo5uvw80k3K
Content-Type: text/abcde
Content-Range: bytes 1-2/5

bc
----cpp-httplib-multipart-data-REkZlTo5uvw80k3K
Content-Type: text/abcde
Content-Range: bytes 4-4/5

e
----cpp-httplib-multipart-data-REkZlTo5uvw80k3K--
`